admin_guide/manage_nodes.adoc

```diff
@@ -245,27 +245,45 @@ To list pods that will be migrated without actually performing the evacuation,
 use the `--dry-run` option:
 
 ----
-$ oadm manage-node <node1> <node2> \
-    --evacuate --dry-run [--pod-selector=<pod_selector>]
+$ oadm drain <node1>  --dry-run=true
```

Comment:
`oadm` has been removed:  openshift/origin#17396
it's being re-added already but with a fat decprecated warning though
`oc adm` is preferred